Src/BootCamp.Chapter/Computer/DesktopComputer.cs

```diff
@@ -1,41 +1,22 @@
 ﻿namespace BootCamp.Chapter.Computer
 {
-    public class DesktopComputer
+    public abstract class DesktopComputer
     {
-        private Body _body;
-        public Body GetBody()
+        protected DesktopComputer()
         {
-            return _body;
-        }
-        
-        private Ram _ram;
-        public Ram GetRam()
-        {
-            return _ram;
-        }
-
-        private Cpu _cpu;
-        public Cpu GetCpu()
-        {
-            return _cpu;
-
-        }
-        private Gpu _gpu;
-        public Gpu GetGpu()
-        {
-            return _gpu;
+            AssemblePC();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this call from a constructor to the overridable 'AssemblePC' method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5149533)